scene/main/scene_main_loop.h

```diff
@@ -350,6 +350,7 @@ class SceneTree : public MainLoop {
 	void quit();
 
 	void set_input_as_handled();
+	bool was_input_handled();
```

Comment:
It looks fine, though I wonder if we shouldn't use `is_input_handled()` as name, to match most other bool accessors which start with `is_`.